charts/egeria-platform/templates/statefulset.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "egeria-platform.fullname" . }}
+  labels:
+    {{- include "egeria-platform.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  serviceName: {{ include "egeria-platform.fullname" . }}
+  selector:
+    matchLabels:
+      {{- include "egeria-platform.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "egeria-platform.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "egeria-platform.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: egeria
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "egeria-platform.fullname" . }}-config
+            {{ if .Values.secretName }}
+            - secretRef:
+                name: {{ .Values.secretName }}
+            {{- end }}
+          ports:
+            - containerPort: {{ .Values.service.platformPort }}
+              name: platform
+          {{ if .Values.egeria.debug }}
+            - containerPort: {{ .Values.service.debugPort }}
+              name: debug
+          {{ end }}
+          # No other checks until this passes
+          startupProbe:
+            exec:
+              command:
+                - stat
+                - /tmp/done.txt
+            initialDelaySeconds: 10
```

Comment:
these seem very aggressive settings. Given the range of environments the charts are run on, I would suggest more relaxed timings (and can add configurability/info to docs on making them more aggressive) - applies t

Reply:
It tries 25 times with a period of 5 seconds, so more than 2min. Dont you think this is enough?